app/assets/stylesheets/active_skin.sass

```diff
@@ -31,54 +31,65 @@ body.active_admin
     padding-left: 130px
     color: $skinTextColor
     text-indent: -999999px
+  .header-item
+    top: 0
```

Comment:
Remove the small gap at the top of the header item. Because ActiveAdmin specifies `top: 3px`.